apps/common/models/file-upload.js

```diff
@@ -29,14 +29,40 @@ module.exports = class UploadModel extends Model {
         }
         return resolve(data);
       });
-    }).then(data => {
-      return this.auth().then(bearer => {
-        data.url = (data.url.replace('/file', '/vault')) + '&token=' + bearer.bearer;
+    }).then(async data => {
+      try {
+        const token = await this.authWithRetries(config.keycloak.authTokenRetries);
+        data.url = (data.url.replace('/file', '/vault')) + `&token=${token}`;
         return data;
-      });
+      } catch (e) {
+        throw e
```

Comment:
Is it worth adding a descriptive error message here?

Reply:
see line 57

Reply:
o ok, never seen that before.  I guess if it catches all cases, that's all good 👍